packages/components/TabList/src/TabList/useTabList.ts

```diff
@@ -72,6 +73,49 @@ export const useTabList = (props: TabListProps): TabListInfo => {
     [setTabKeys],
   );
 
+  const incrementTabKey = React.useCallback(
+    (increment: number) => {
+      if (increment === 0) {
+        return;
+      }
+
+      const currentIndex = tabKeys.indexOf(selectedTabKey);
+
+      if (currentIndex === -1 && __DEV__) {
+        console.error(`Selected tab key does not exist in tab key array. Tab Key: ${selectedTabKey}. Keys: ${tabKeys}`);
+      }
```

Comment:
Should we keep this, or just remove it?

Reply:
i would say that if the API/function isn't exposed externally, it's a bit overkill to check for weird inputs since you should be able to know exactly what is coming... but not a strong opinion.

Reply:
Went ahead and removed the log.